QuickDeployment/QuickDeploymentSetup.md

```diff
@@ -127,18 +138,19 @@ See [[QDE Client Setup|QuickDeploymentClientSetup]].
 
 ## Database Password Changes
 
-The database credentials are currently pre-set. If you would like to change the credentials associated with the database, you will need to follow these steps.
+The database credentials are currently pre-set.
+If you would like to change the credentials associated with the database, you will need to follow these steps.
 
-### 1. Change the database access credentials
+### Change the database access credentials
 
-### 2. Reinstall the chocolatey-management-service package
+### Reinstall the chocolatey-management-service package
 
 ```powershell
 choco uninstall chocolatey-management-service -y
 choco install chocolatey-management-service -y --package-parameters-sensitive=”’/ConnectionString=””Server=localhost\SQLEXPRESS;Database=ChocolateyManagement;User ID=ChocoUser;Password=NewPassword;””’”
 ```
 
-### 3. Reinstall the chocolatey-management-web package
+### Reinstall the chocolatey-management-web package
```

Comment:
Since these are steps through, change them to steps as an ordered list and take away the headings.